<?xml version="1.0"?> 
<launch>
    <arg name="node_name" default="bno055" />
    <arg name="serial_port" default="/dev/ttyUSB0" />
    <arg name="operation_mode" default="IMU" />

    <node pkg="bno055" type="bno055_calibrate.py" name="$(arg node_name)_calibrate" output="screen">
        <param name="serial_port" value="$(arg serial_port)" />    
        <param name="operation_mode" value="$(arg operation_mode)" />    
    </node>

</launch>
